ansible/README.md

```diff
@@ -1,11 +1,32 @@
 # Ansible
 
 This directory contains all [Ansible](http://docs.ansible.com/) playbooks and
-files to setup, configure and deploy services automatically.
+files required to setup services automatically.
+
+## Installing Ansible
+
+If you still do not have Ansible installed in your system, setup EPEL yum
+repository and run:
+
+`yum install -y ansible`
 
 ## Running Ansible playbooks
 
-You can run Ansible playbooks by executing `ansible-playbook` command:
+First, update the host in the hosts.ini section which corresponds to the
+playbook which will be executed (eg if you will execute the jenkins-master
+playbook, update the host in [jenkins-master] section in hosts.ini)
+
+Then, update the variables values in vars.yaml. If you are creating a production
+Jenkins server, provide the production GitHub organization (open-power-host-os) in
+github_organization_name and ghprob_admin_organization variables. If you are creating
+a test Jenkins server, provide your GitHub organization and user name in
+github_organization_name, ghprb_admin_user and ghprb_admin_organization
+variables.
```

Comment:
This looks like the user should use the GitHub organization name in the `github_organization_name` variable, which is not true for our custom setups. I suggest:
"you may provide your user name in github_organization_name and ghprb_admin_user variables and leave ghprb_admin_organization empty."